pkgs/development/compilers/mkcl/default.nix

```diff
@@ -57,6 +57,7 @@ stdenv.mkDerivation rec {
   enableParallelBuilding = true;
 
   meta = with lib; {
+    mainProgram = "mkcl";
```

Comment:
all else being equal, we tend to sort these attribute sets alphabetically. not a strict requirement but while you're editing the commit message consider moving this to be between `platforms` and `maintainers` & same for the other files.